spec/models/date_range_spec.rb

```diff
@@ -17,4 +17,9 @@
     negativeDates = FactoryGirl.build(:date_range, :with_negative_range)
     expect(negativeDates).to_not be_valid
   end
+
+  it "should print only one date if it is on a single day" do
+    date_range = FactoryGirl.build :date_range, :on_single_day
+    expect(date_range.to_s).to eq(I18n.l(Date.tomorrow))
```

Comment:
This relies really strongly on the `on_single_day` trait returning the next day. Maybe compare to localization of just the start date?